src/CommunityToolkit.Maui/Views/BaseTemplatedView.shared.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace CommunityToolkit.Maui.Views;
+
+/// <summary>
+/// Abstract class that templated views should inherit
+/// </summary>
+/// <typeparam name="TControl">The type of the control that this template will be used for</typeparam>
+public abstract class BaseTemplatedView<TControl> : TemplatedView where TControl : View, new()
```

Comment:
TemplatedView uses Compatibility StackLayout. I think we shouldn't use controls from Compatibility namespace